src/apps/mmfinance/cronos/MMfinance.auto-mmf.contract-position-fetcher.ts

```diff
@@ -0,0 +1,54 @@
+import { Inject } from '@nestjs/common';
```

Comment:
Please lowercase the `mmfinance` in all of the filenames (we want it to match the name of the directory `mmfinance` for consistency)... i.e.: `src/apps/mmfinance/cronos/mmfinance.auto-mmf.contract-position-fetcher.ts`.